### PR TITLE
Add extension ".wl" for wolfram-mode

### PR DESCRIPTION
**Pull request by kenkangxgwe**
I just associated `.wl` with `wolfram-mode` in the `packages.el` of `extra-langs` layer.

### CODE REVIEW

File: layers/+lang/extra-langs/packages.el [hunk 1]
```diff
@@ -36,8 +36,8 @@
 (defun extra-langs/init-thrift ()
   (use-package thrift :defer t))
 
-;; no associated extension because conflicts with more common Objective-C, manually invoke for .m files.
```
Comment (bmag) [inline layers/+lang/extra-langs/packages.el]: modify this comment, but don't delete it entirely. The info about .m files is important

**Comment by bmag** [comment]
Is it normal to associate `.wl` files with `wolfram-mode`? I couldn't find any files that use the `.wl` extension...

**Comment by kenkangxgwe** [comment]
Hi @bmag , please just check the [Wolfram Language on Wikipedia](https://en.wikipedia.org/wiki/Wolfram_Language) and [Wolfram Language Scripts](https://reference.wolfram.com/language/tutorial/WolframLanguageScripts.html). You will see this filename extension. Unlike .m and .nb, .wl is preferred for it is distinguishable from MATLAB and Objective-C file, and more human readable than notebook format.

**Comment by bmag** [comment]
Thank you :+1: Cherry-picked into develop, you can safely delete your branch. I also squashed the two commits into one commit, and broke the long comment line in two in https://github.com/syl20bnr/spacemacs/commit/f853b78148b4985bcc4e6c1232564406ee9c4466. Next time please remember to do so.

**Pull request closed without merging**